OpenRA.Mods.Common/Traits/AutoTarget.cs

```diff
@@ -24,8 +24,11 @@ public class AutoTargetInfo : ITraitInfo, Requires<AttackBaseInfo>
 		[Desc("Set to a value >1 to override weapons maximum range for this.")]
 		public readonly int ScanRadius = -1;
 
-		[Desc("Possible values are HoldFire, ReturnFire, Defend and AttackAnything.")]
-		public readonly UnitStance InitialStance = UnitStance.AttackAnything;
+		[Desc("Possible values are HoldFire, ReturnFire, Defend and AttackAnything. Used for computer-controlled players.")]
```

Comment:
Hmm, like this? Not sure what kind of clarification you're asking for. I updated the `Desc` and renamed the variable to `InitialStanceAI`.